corehq/apps/app_manager/forms.py

```diff
@@ -111,13 +134,30 @@ def __init__(self, *args, **kwargs):
 
         self.helper.label_class = 'col-sm-3 col-md-2'
         self.helper.field_class = 'col-sm-9 col-md-8 col-lg-6'
+        self.helper.field_style = 'ganggs'
```

Comment:
what does `ganggs` do?

Reply:
Oops, removed now. Was testing something, but didn't remove this.